guides/release/routing/defining-your-routes.md

```diff
@@ -64,7 +64,7 @@ Router.map(function() {
 
 The route defined above will by default use the `blog-post.js` route handler,
 the `blog-post.hbs` template, and be referred to as `blog-post` in any
-`{{link-to}}` helpers.
+`<LinkTo>`.
```

Comment:
`<LinkTo>` components. (or helpers? but I thought they were components).

Reply:
I normalized to component where I saw. In this case I removed the qualifier on purpose, as it doesn't seem necessary for understanding.